01-node-tutorial/answers/12-http.js

```diff
@@ -0,0 +1,19 @@
+/* HTTP MODULE */
+
+const http = require("http"); //create http server
+
+const server = http.createServer((req, res) => {
+  if (req.url === "/") {
+    console.log("Welcome to my pg!, I am learning Node");
```

Comment:
I think you want to use `res.end` here instead of `console.log`. `res.end` will send the data back to the browser, while `console.log` just prints the message to the console on the server (doesn't send any data to the browser).